build/common/installer/scripts/fluent-bit-conf-customizer.rb

```diff
@@ -211,6 +227,15 @@ def substituteFluentBitPlaceHolders
     File.open(@fluent_bit_common_conf_path, "w") { |file| file.puts new_contents }
     puts "config::Successfully substituted the placeholders in fluent-bit-common.conf file"
 
+    puts "config::Starting to substitute the placeholders in fluent-bit-network-flow-logs.conf file for log collection"
+    network_flow_logs_conf_path = "/etc/opt/microsoft/docker-cimprov/fluent-bit-network-flow-logs.conf"
+    if File.exist?(network_flow_logs_conf_path)
```

Comment:
we dont need to execute the code when networkFlowLogsEnabled != true, lets add check for networkFlowLogsEnabled

Reply:
added